### ISSUE TITLE
Do not run commented line of code in a scrapbook

### ISSUE THREAD

**Issue by johnpapa** [Microsoft/vscode-cosmosdb#202]
- When I comment out a command in a scrapbook, and then execute the line, it still runs. I would expect that a commented out line in scrapbook would not execute

**Comment by PrashanthCorp** [comment]
Do you have any steps to repro this? 

Here's what I tried : 
Case 1 : 1 line, commented. 
`//db.coll.find()`
Running this gives me a "No executable command found." error

Case 2 : Commented line of code, followed by uncommented line
`//db.coll.count()`
`db.coll.find()`
Place cursor on the first line. 
It does not run the count command, it runs the find command. 
To verify this, I place a breakpoint at [this line](https://github.com/Microsoft/vscode-cosmosdb/blob/0c37c49ae04303fda0fb075e0c96a17e5d968c1b/src/mongo/commands.ts#L34) and verify that the `find` block gets executed. 

Hoping to hear back about your repro scenario. 

PS : I'm running on the current master (at [this point](https://github.com/Microsoft/vscode-cosmosdb/tree/0c37c49ae04303fda0fb075e0c96a17e5d968c1b) in the code history)

**Comment by johnpapa** [comment]
I did some more digging ... notice the GIF below. Here I have 2 commands, and i put my cursor on the second one and execute it. That works. Then i comment the second one and keep my cursor on that line and execute, and it runs the line before it.

![2017-12-12_09-01-54](https://user-images.githubusercontent.com/1202528/33888541-5b9ac916-df1b-11e7-8cad-4cec8cac85a6.gif)


i could no longer get a commented line to execute when my cursor was on it. This must have been fixed or I made a mistake before.
